nss-day-cohort-60/django-taters-api
# Lh comments

Part of PR that will close ticket 122


# Description of Proposed Changes
added an author serializer to comment_view

---


# Steps to Test

1. Fetch the lh-comments branch and switch to it
2. Start/restart debugger
3. Open postman and make a GET request http://localhost:8000/comments to verify that author is expanded
